services/wafv2.py

```diff
@@ -2,28 +2,83 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("wafv2", region_name="us-east-1")
+
+cloudfront_client = boto3.client("cloudfront", "us-east-1")
 
 
 def wafv2_logging_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    webacls = client.list_web_acls(Scope="REGIONAL")["WebACLs"]
+
+    for webacl in webacls:
+        print(webacl["ARN"])
+        configuration = client.get_logging_configuration(ResourceArn=webacl["ARN"])
+        if configuration["LoggingConfiguration"] != []:
+            compliant_resources.append(webacl["ARN"])
+        else:
+            non_compliant_resources.append(webacl["ARN"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def wafv2_rulegroup_logging_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    rule_groups = client.list_rule_groups(Scope="REGIONAL")["RuleGroups"]
+
+    for rule_group in rule_groups:
+        configuration = client.get_rule_group(ARN=rule_group["ARN"])
+        if configuration["RuleGroup"]["VisibilityConfig"]["CloudWatchMetricsEnabled"] == True:
+            compliant_resources.append(rule_group["ARN"])
+        else:
+            non_compliant_resources.append(rule_group["ARN"])
```

Comment:
CloudFront용 rule group 추가
```suggestion
    regional_rule_groups = client.list_rule_groups(Scope="REGIONAL")["RuleGroups"]
    cloudfront_rule_groups = global_client.list_rule_groups(Scope="CLOUDFRONT")["RuleGroups"]


    for rule_group in regional_rule_groups:
        configuration = client.get_rule_group(ARN=rule_group["ARN"])
        if configuration["RuleGroup"]["VisibilityConfig"]["CloudWatchMetricsEnabled"] == True:
            compliant_resources.append(rule_group["ARN"])
        else:
            non_compliant_resources.append(rule_group["ARN"])

    for rule_group in cloudfront_rule_groups:
        configuration = global_client.get_rule_group(ARN=rule_group["ARN"])
        if configuration["RuleGroup"]["VisibilityConfig"]["CloudWatchMetricsEnabled"] == True:
            compliant_resources.append(rule_group["ARN"])
        else:
            non_compliant_resources.append(rule_group["ARN"])
```